### PR TITLE
Adjusting "Shirakami Fubuki" title.

### PR DESCRIPTION
**Pull request by bempfoobq052021**
Changes in custom.css : .presentation-title, .footer-nav line-height (accomodating mobile version).
Changes in index.html : 
```
<h1> to <span> (accomodating mobile version) 
style=font moved to custom.css
line-height added to adjust <span> to look more natural like <h1>
```

### CODE REVIEW

File: assets/css/custom.css [hunk 1]
```diff
@@ -15,12 +15,32 @@ video {
     left: 0;
 }
 
+@media screen and (max-width: 768px) {
+  video {
+	min-width: 320px;
```
Comment (nevrending) [inline assets/css/custom.css]:
```suggestion
    min-width: 320px;
```

File: assets/css/custom.css [hunk 2]
```diff
@@ -15,12 +15,32 @@ video {
...
     z-index: 3;
 }
 
 .presentation-title {
     font-weight: 500;
+	font-size: 5.5rem;
+	background: linear-gradient(180deg,#fff 35%,#8fb2c3);
+	-webkit-background-clip: text;
+	-webkit-text-fill-color: transparent;
+	line-height: 6rem;
```
Comment (nevrending) [inline assets/css/custom.css]:
```suggestion
    font-size: 5.5rem;
    background: linear-gradient(180deg,#fff 35%,#8fb2c3);
    -webkit-background-clip: text;
    -webkit-text-fill-color: transparent;
    line-height: 6rem;
```

File: assets/css/custom.css [hunk 3]
```diff
@@ -15,12 +15,32 @@ video {
...
+}
+
+@media screen and (max-width: 767px) {
+  .presentation-title {
+	font-size: 3.5rem;
+	line-height: 5rem;
```
Comment (nevrending) [inline assets/css/custom.css]:
```suggestion
    font-size: 3.5rem;
    line-height: 5rem;
```

File: index.html [hunk 4]
```diff
@@ -71,7 +71,7 @@
     <div class="content-center">
       <div class="container">
         <div class="title-brand" style="max-width: 960px">
-          <h1 class="presentation-title" style="font-size: 7em">Shirakami Fubuki</h1>
+          <span class="presentation-title">Shirakami Fubuki</h1>
```
Comment (nevrending) [inline index.html]: Tag has not been closed properly.

```suggestion
          <span class="presentation-title">Shirakami Fubuki</span>
```

**Review by nevrending — CHANGES_REQUESTED** [review]
Hi there! Thanks for your PR. It makes the mobile view looks better.

image

Just a couple of things:
- We use space for indentation instead of tabs
- The `span` tag that replaced `h1` hasn't been closed
- Footer CSS fixed mobile, but makes desktop looks misaligned (see below).

image

I've put in some inline comments that you can directly commit or you can make a commit that contains all of them, whichever works best for you.

File: index.html (hunk 4 above)
Comment (bempfoobq052021) [reply index.html]: Whoops. I'll accept this immediately.

File: assets/css/custom.css (hunk 3 above)
Comment (bempfoobq052021) [reply assets/css/custom.css]: Looks good to me~! Thanks for the fix!

File: assets/css/custom.css (hunk 2 above)
Comment (bempfoobq052021) [reply assets/css/custom.css]: Looks good to me~!

**Comment by bempfoobq052021** [comment]
On this problem : 
Footer CSS fixed mobile, but makes desktop looks misaligned (see below).

Fixed by changing `<span class="copyright"> to <p class="copyright" style="line-height:20pt">`
I use &nbsp for the spaces though.

File: index.html [hunk 5]
```diff
@@ -233,18 +233,16 @@ <h2 class="title">Community</h2>
...
         </div>
       </div>
     </div>
   </footer>
   <span id="sideimg-left"></span>
   <span id="sideimg-right"></span>
   <script src="assets/js/core/jquery.min.js"></script>
-  <script src="assets/js/core/popper.min.js"></script>
-  <script src="assets/js/core/bootstrap.min.js"></script>
-  <script src="assets/js/theme.min.js?v=2.2.0"></script>
```
Comment (nevrending) [inline index.html:248]: These should not be removed as they enabled the dropdown menu and stuffs.

**Review by nevrending — APPROVED** [review]
I'll accept this PR for now, since the hero fix is good, and work on the remaining issues on a separate commit.

Thanks again for your contribution!

**Pull request merged**